addons/l10n_vn/__manifest__.py

```diff
@@ -19,11 +19,12 @@
     - Trobz
 """,
     'depends': [
-        'account',
```

Comment:
please don't remove this. because the l10n_vn is also use for Vietnamese accounting, so it's must depend on account.

Reply:
Just FYI, account_qr_code_emv depends on account so this module will still depend on account indirectly too. It doesn't change this :)